src/mpid/ch4/src/ch4_globals.c

```diff
@@ -95,7 +95,7 @@ int MPID_Abort(MPIR_Comm * comm, int mpi_errno, int exit_code, const char *error
     MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPIDI_ABORT);
     fflush(stderr);
     fflush(stdout);
-    if (MPIR_Comm_size(comm) == 1)
+    if (comm && MPIR_Comm_size(comm) == 1)
```

Comment:
Do we want to use `||` instead to limit the scope?

Reply:
The complaint from Coverity was that `comm` might be `NULL`, so `||` wouldn't solve that problem.

Reply:
How about `if (comm == NULL || MPIR_COMM_size(comm) == 1)`. Evaluating left to right should guarantee we don't dereference `NULL`.

Reply:
Yeah, that's probably fine. I think the point here was that if there's only one process, it shouldn't call `PMI_Abort`. Maybe it should check `MPIR_Comm_size(MPIR_Process.comm_world) == 1`?

Reply:
`PMI_Abort` has no scope capabilities. Think of my proposal as effectively aborting only `MPI_COMM_SELF` when there is no associated comm/window/file.

Reply:
Done.